docs/book/src/reference/formatting-pipeline.md

```diff
@@ -0,0 +1,64 @@
+# The formatting pipeline
+
+## Query matching
+
+The first step is to apply the formatting queries against an input. That
```

Comment:
Rather than just "the first step", this could be "the heart of Topiary" or something along those lines.

Reply:
Good idea. I call it "the essence of Topiary" elsewhere, so I'll go with that.